lib/membrane_rtc_engine/endpoints/rtsp_endpoint.ex

```diff
@@ -31,26 +56,56 @@ if Enum.all?(
                 source_uri: [
                   spec: URI.t(),
                   description: "URI of source stream"
+                ],
+                rtp_port: [
+                  spec: pos_integer() | nil,
```

Comment:
Unsure what would be best to put here as the default value: `nil`, `:any`, or simply `0`?

`UDP.Source` uses `0` to designate "any available port", but I believe `:any` or `nil` would make more sense to the end-user than `0`